tests/test_sensitivity.py

```diff
@@ -30,6 +39,54 @@
 ]
 
 
+class SobolResult:
+    def __init__(self, sobol_dict):
+        self.sobol_dict = sobol_dict
+
+    def to_df(self):
+        df = [
+            pd.DataFrame(
+                self.sobol_dict,
+                index=[f"param{i + 1}" for i in range(len(self.sobol_dict["ST"]))],
+            )
+        ]
+        return df
+
+
+def sobol_res_mock():
+    sobol_dict = {
+        "S1": [-2.5, 0.8, 3],
+        "S1_conf": [1e-2, 1e-2, 1e-2],
+        "ST": [0.5, 0.7, 0.15],
+        "ST_conf": [1e-2, 3e-2, 5e-2],
+    }
+    return SobolResult(sobol_dict)
+
+
+class MorrisResult:
+    def __init__(self, morris_dict):
+        self.morris_dict = morris_dict
+
+    def to_df(self):
+        df = pd.DataFrame(
+            self.morris_dict,
+            index=[f"param{i + 1}" for i in range(len(self.morris_dict["mu"]))],
+        )
+        return df
+
+
+def morris_res_mock():
+    morris_dict = {
+        "mu": [0.2, 0.5, 0.1],
+        "mu_star": [0.25, 0.55, 0.15],
+        "mu_star_conf": [0.025, 0.055, 0.015],
+        "sigma": [0.05, 0.1, 0.02],
+        "euclidian distance": [0.5, 0.7, 0.15],
+        "normalized euclidian distance": [0.1, 0.2, 0.05],
+    }
+    return MorrisResult(morris_dict)
```

Comment:
A bit complicated.
Maybe we should simplify and gather hard coded, or configuration dict at the begining of the module